.gitignore

```diff
@@ -12,3 +12,5 @@ html/
 doxygen/doxypy-0.4.2/
 .pytest_cache/
 .vscode/
+
+pytest.ini
```

Comment:
why do you want to ignore pytest.ini?

Reply:
so that when you put your custom configuration there, it would not be staged for commit - for example cassandra_dir